pkg/apis/componentconfig/types.go

```diff
@@ -712,3 +712,15 @@ type PersistentVolumeRecyclerConfiguration struct {
 	// in a multi-node cluster.
 	IncrementTimeoutHostPath int32 `json:"incrementTimeoutHostPath"`
 }
+
+const (
+	ApiServerConfigMapName string = "apiserver-config"
```

Comment:
apiserver should expect its configmap to appear as a file on disk. It should have a flag expressing the location.